harness/attributes/docker-01-base.yml

```
@@ -24,7 +24,7 @@ attributes.default:
     memcached:
       image: memcached:1-alpine
     mongodb:
-      image: mongo:4.4
+      image: mongo:7.0
```

This will need to be mentioned in the upgrade notes as I don't think you can jump from 4.4 to 7.0 without the intermediate steps.

---

```suggestion
      image: mongo:5.0
```

5.0 still around for a year so this will simplify the upgrade process, and allow local dev to upgrade automatically

I'm adding a separate PR for release notes for both this and postgres